CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ option(LIB_PROTO_MUTATOR_TESTING "Enable test building" ON)
 option(LIB_PROTO_MUTATOR_DOWNLOAD_PROTOBUF
        "Automatically download working protobuf" OFF)
 option(LIB_PROTO_MUTATOR_WITH_ASAN "Enable address sanitizer" OFF)
+option(LIB_DIR "Directory to install libraries" "lib")
```

Comment:
Why do we need LIB_DIR at all?
Can we just use CMAKE_INSTALL_PREFIX?

Reply:
When generating the pkg-conf .pc file for the library, we need a way of knowing if the libraries should be linked from /usr/lib, /usr/lib64, /usr/lib32, etc.

Reply:
So why can't we use CMAKE_INSTALL_PREFIX (or CMAKE_INSTALL_PREFIX/lib) for that?

Reply:
I think the way it is fixed in this CL is just a common way some distros handle the lib directory.

This is the change I was going to submit: https://github.com/Allen-Webb/libprotobuf-mutator/commit/8c367920f451f3f7bb26a5f2be36be89dae89d99 which simply moves the variable out of the inf statement.

Reply:
I've pulled Allen-Webb@8c36792 as LIB_DIR is definitely ended up in the wrong place.
I see why we need LIB_SUFFIX there.
I am not sure why we need LIB_DIR as an option.

Reply:
You are right, the fix of @Allen-Webb is better, I dropped my commit.